dev/NavigationView/NavigationView_InteractionTests/CommonTests.cs

```diff
@@ -0,0 +1,1592 @@
+                        {
+                            using (var waiter = isPaneOpenCheckBox.GetToggledWaiter())
+                            {
+                                isPaneOpenCheckBox.Toggle();
+                                waiter.Wait();
+                            }
+                        }
+
+                        Button navButton = new Button(FindElement.ById("TogglePaneButton"));
+                        Verify.AreEqual(320, navButton.BoundingRectangle.Width);
+                    }
+                }
+            }
+        }
+
+        [TestMethod]
+        [TestProperty("Description", "Ensure that the NavigationView button is rendering as expected if it's targeting RS3")]
+        public void VerifyShouldPreserveNavigationViewRS3Behavior()
+        {
+            using (var setup = new TestSetupHelper(new[] { "NavigationView Tests", "NavigationView PreserveRS3 Test" }))
+            {
+                if (!PlatformConfiguration.IsOsVersionGreaterThanOrEqual(OSVersion.Redstone4))
+                {
+                    Log.Warning("This test is only designed to run on RS4+ machines");
+                    return;
+                }
+
+                CheckBox isPaneOpenCheckBox = new CheckBox(FindElement.ById("IsPaneOpenCheckBox"));
+
+                // On phone, the pane will initially be in the closed compact state, so open it before
+                // proceeding with the test.
+                if (isPaneOpenCheckBox.ToggleState == ToggleState.Off)
+                {
+                    using (var waiter = isPaneOpenCheckBox.GetToggledWaiter())
+                    {
+                        isPaneOpenCheckBox.Toggle();
+                        waiter.Wait();
+                    }
+                }
+
+                // In RS4 or late application, togglePaneTopPadding is 0 when ExtendViewIntoTitleBar=true, 
+                // but for RS3 application, we expected it be not 0 because apps like Wallpaper make use of it
+                var result = new Edit(FindElement.ById("TestResult"));
+                using (var waiter = new ValueChangedEventWaiter(result))
+                {
+                    Button button = new Button(FindElement.ById("GetTopPaddingHeight"));
+                    button.Invoke();
+                    waiter.Wait();
+                }
+                var togglePaneTopPadding = Convert.ToInt32(result.Value);
+                Verify.AreNotEqual(0, togglePaneTopPadding);
+
+                // TestFrame is disabled before the testcase. we should enable it and prepare for next test case
+                var testFrame = new CheckBox(FindElement.ById("TestFrameCheckbox"));
+                testFrame.Check();
+                Wait.ForIdle();
+            }
+        }
+
+        [TestMethod]
+        [TestProperty("Description", "Temporary bootstrapping test, can be retired once Horizontal Nav View is out of incubation")]
+        public void EnsureNoCrashesInHorizontalFlipMenuItems()
```

Comment:
>EnsureNoCrashesInHorizontalFlipMenuItems [](start = 20, length = 40)

I think these would both be covered by other tests and could probably be deleted